### PR TITLE
Fix building media sharing tests

### PR DESCRIPTION
**Pull request by gwawiork**
Fix issue #387

### CODE REVIEW

File: test_extensions/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Remember curren source directory (`test_conformance').
+set( CLConf_Install_Base_Dir "${CMAKE_CURRENT_SOURCE_DIR}" )
```
Comment (kpet) [inline test_extensions/CMakeLists.txt]: Where is this used?

File: test_extensions/media_sharing/procs.h [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+
+#ifndef __PROCS_H__
```
Comment (kpet) [inline test_extensions/media_sharing/procs.h]: `__PROCS_H__` is a reserved identifier. See https://en.cppreference.com/w/c/language/identifier. Please use `#pragma once` or `MEDIA_SHARING_PROCS_H` for example.

File: test_extensions/CMakeLists.txt (hunk 1 above)
Comment (gwawiork) [reply test_extensions/CMakeLists.txt]: I've removed this

File: test_extensions/media_sharing/procs.h (hunk 2 above)
Comment (gwawiork) [reply test_extensions/media_sharing/procs.h]: fixed

Comment (kpet) [reply test_extensions/media_sharing/procs.h]: You missed the point :). The problem was the double underscore followed by a capital letter...

**Pull request merged**